docs/resources/Channel.md

```diff
@@ -238,15 +238,6 @@ Represents a message sent in a channel within Discord.
 | type      | integer | [type of message activity](#DOCS_RESOURCES_CHANNEL/message-object-message-activity-types)                          |
 | party_id? | string  | party_id from a [Rich Presence event](#DOCS_RICH_PRESENCE_HOW_TO/updating-presence-update-presence-payload-fields) |
 
-###### Message Application Structure
-
-| Field        | Type      | Description                   |
-| ------------ | --------- | ----------------------------- |
-| id           | snowflake | id of the application         |
-| cover_image? | string    | id of the embed's image asset |
-| description  | string    | application's description     |
-| icon         | ?string   | id of the application's icon  |
-| name         | string    | name of the application       |
```

Comment:
Why was this removed?

Reply:
The idea was that rather than keep the partially duplicated application object here (which lead to some inconsistencies like the cover image and icon hashes being called "id"s), it'd be more consistent to link to the application object in the oauth2 section, lable this instance as" partial" and show the fields that are included in the partial application here in an example.